src/App.js

```diff
@@ -1,20 +1,35 @@
 import React, { Component } from 'react';
-import { BrowserRouter as Router, Route } from 'react-router-dom';
-import './App.css';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'; import './App.css';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import { install } from 'redux-loop';
 import MovieSearch from './pages/MovieSearch';
 import ErrorBoundary from './components/error-boundary/ErrorBoundaty';
 import MovieSelected from './pages/MovieSelected';
+import Footer from './components/footer/Footer';
+import rootReducer from './store/reducers';
+import PageNotFoundComponent from './pages/PageNotFoundComponent';
+
+const store = createStore(rootReducer, install());
```

Comment:
It would be much cleaner and easier for code reading to move store creation to a separate file.